addon/Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG BUILD_VERSION
+FROM yp87/leaf2mqtt:${BUILD_VERSION} AS build
+FROM alpine
```

Comment:
you could use existing home assistant add-on base image that already contains bashio: https://github.com/home-assistant/docker-base

Reply:
Will it really make a difference? To me it seems easier to maintain and it would make sure we use the latest version of bashio and  no dependencies or anything else is missing

Reply:
It's mostly the s6 layer that annoys me, and I know I will have to update my bashio code for the latest version, but if that's a deal breaker I'll sort it out.

Reply:
Done.